sklearn/utils/version.py

```diff
@@ -0,0 +1,361 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
You should add a module docstring here. Among other things, it should mention where (which package, file, version) the code has been taken from.